src/components/Navbar.vue

```diff
@@ -22,6 +22,8 @@
               <font-awesome-icon icon="graduation-cap" fixed-width/> Javadocs</a></li>
             <li><a href="https://forums.spongepowered.org/c/plugins/plugin-releases">
               <font-awesome-icon icon="plug" fixed-width/> Plugins</a></li>
+            <li><a href="https://ore.spongepowered.org/">
+              <font-awesome-icon icon="cubes" fixed-width/> Ore <b>Beta</b></a></i>
```

Comment:
needs to be `<font-awesome-icon icon="cubes" fixed-width/> Ore <b>Beta</b></a></li>`

Reply:
fixed